src/ucp/core/ucp_context.c

```diff
@@ -522,6 +529,12 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "Prefetch memory when using global VA MR",
    ucs_offsetof(ucp_context_config_t, gva_prefetch), UCS_CONFIG_TYPE_BOOL},
 
+  {"FORCE_REQUEST_FLAGS", "",
+   "Defines forced request flags. Possible flags are: no_imm_cmpl, fast_cmpl, "
+   "force_imm_cmpl, multi_send",
```

Comment:
I think UCS_CONFIG_TYPE_BITMAP already prints the list of possible values

Reply:
`UCX_MEMTYPE_REG_WHOLE_ALLOC_TYPES` uses `UCS_CONFIG_TYPE_BITMAP` as well, so it is consistent with the format used for other configuration options in the table.
`ucx_info -f`:
![image](https://github.com/user-attachments/assets/2172e7af-3e09-4396-b36c-7de69936e1d5)

for comparison:
![image](https://github.com/user-attachments/assets/b12b7a8c-77e5-4316-b99f-2dd9b4d6537e)